src/Geopilot.Api/Controllers/MandateController.cs

```diff
@@ -77,6 +77,30 @@ public async Task<IActionResult> Get(
         return Ok(result);
     }
 
+    /// <summary>
+    /// Gets the mandate with the specified <paramref name="id"/>.
+    /// </summary>
+    /// <returns>The mandate.</returns>
+    [HttpGet("{id}")]
+    [Authorize(Policy = GeopilotPolicies.Admin)]
+    [SwaggerResponse(StatusCodes.Status200OK, "Returns the mandate", typeof(Mandate), "application/json")]
+    [SwaggerResponse(StatusCodes.Status401Unauthorized, "The current user is not authorized to get this mandate.")]
```

Comment:
PP: etwas inkonsequent, da /mandate für den User ja alle seine verfügbaren liefert und hier
```suggestion
    [SwaggerResponse(StatusCodes.Status401Unauthorized, "The current user is not authorized to get individual mandates.")]
```